features/custom_matchers/define_block_matcher.feature

```diff
@@ -42,3 +42,28 @@ Feature: define a matcher supporting block expectations
       """
     When I run `rspec ./block_matcher_spec.rb`
     Then the example should pass
+
+  Scenario: define a block matcher using shortcut
+    Given a file named "block_matcher_spec.rb" with:
+      """ruby
+      RSpec::Matchers.define :support_blocks_with_errors do
+        match(:notify_expectation_failures => true) do |actual|
+          actual.call
+          true
+        end
+
+        supports_block_expectations
+      end
+
+      RSpec.describe "a custom block matcher" do
+        specify do
+          expect {
+            expect {
+              raise RSpec::Expectations::ExpectationNotMetError.new('original')
+            }.to support_blocks_with_errors
+          }.to raise_error(RSpec::Expectations::ExpectationNotMetError, /original/)
+        end
+      end
+      """
+    When I run `rspec ./block_matcher_spec.rb`
+    Then the example should pass
```

Comment:
I don't think this scenario works very well as documentation -- it's written very similarly to our specs but doesn't approximate real usage.  A few things I'd recommend:

* Don't use `expect { }.to raise_error(RSpec::Expectations::ExpectationNotMetError)` -- instead, change the cucumber step from `Then the example should pass` to `Then it should fail with....` (or whatever the wording is for that).  After all, when users are using this feature they're not going to be wrapping things with `expect { }.to raise_error(RSpec::Expectations::ExpectationNotMetError)`.
* Don't use `raise RSpec::Expectations::ExpectationNotMetError.new('original')` -- instead just use a failing expectation in there.  Again, users don't write `raise RSpec::Expectations::ExpectationNotMetError.new('original')` and using it here obscures things, I think.